AndelaOSP/andela-societies-frontend
# #159032573 Enable success ops member to request additional details on a logged activity

#### What does this PR do?
It adds the feature to enable a  logged in `success ops` member to request more/additional details on a logged activity.

#### Description of the task completed?
Have a `Comment` button on the activity cards which a logged in `success ops` member can click and request additional details by filling in the `comments` section in the pop up modal.

#### How can this be tested?
Log into the application as a `success ops` member, on getting to the `Verify activities` page, you should see a `Comment` button. Use this to log your comment on the activity.

#### What are the relevant pivotal tracker stories?
[159032573](https://www.pivotaltracker.com/story/show/159032573)
#### Screenshots 
<img width="1440" alt="screen shot 2018-07-23 at 23 35 12" src="https://user-images.githubusercontent.com/29278184/43101461-1d0d6e04-8ed1-11e8-9fb2-a4ed78686a65.png">
<img width="1440" alt="screen shot 2018-07-23 at 23 35 20" src="https://user-images.githubusercontent.com/29278184/43101466-1ff8c852-8ed1-11e8-84d5-e9485c47bc51.png">
